recipes/discord/index.js

```diff
@@ -1,8 +1,5 @@
 module.exports = Ferdium => class Discord extends Ferdium {
   overrideUserAgent() {
-    const useragent = window.navigator.userAgent;
-    const parts = useragent.split('(KHTML, like Gecko)');
-
-    return parts.join('(KHTML, like Gecko) discord/0.0.250').replace('Electron', 'Discord').replace('Ferdium', 'Discord');
+    return window.navigator.userAgent.replace("Apple Mac OS X", "Intel Mac OS X");
```

Comment:
You will probably also want the older string-replaces in place. Please add your `.replace()` calls as additive - so that other OSes, chip-architectures, etc continue to work.

Reply:
@kishaningithub - I have made the required changes (including bumping up the recipe version number) - please verify once.

Reply:
Tested it in my ARM M1 mac.. The change works flawlessly..